src/utils/filterPeople.tsx

```diff
@@ -0,0 +1,68 @@
+import { Person } from '../types';
+
+export const filterPeople = (
+  people: Person[],
+  sex: string,
+  query: string,
+  centuries: string[],
+  sort: string,
+  order: string,
+) => {
+  let filteredPeople = [...people];
+
+  if (query && query.trim()) {
+    const queryToLower = query.toLocaleLowerCase();
```

Comment:
Consider using `toLowerCase()` instead of `toLocaleLowerCase()` unless you specifically need locale-aware case conversion. This can improve performance slightly if locale-specific behavior is not required.

Reply:
одруківка(((